weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/CompareEvaluationsPage.tsx

```diff
@@ -185,6 +225,23 @@ const CompareEvaluationsPageInner: React.FC<{
     Object.keys(state.loadableComparisonResults.result?.resultRows ?? {})
       .length > 0;
   const resultsLoading = state.loadableComparisonResults.loading;
+
+  // Check if we should show the traceCalls UI
+  const isTraceCallsPath = props.traceCalls && props.traceCalls.length > 0;
+
+  if (isTraceCallsPath) {
+    // Use the new TraceCallsCompareEvaluationsPage component
+    return (
```

Comment:
Verified that current evals meet this condition, meaning all evals use this new view (which is breaks for current evals and is a visual regression). For expediency, let's land the python code separate from this